docs/compiler.md

```diff
@@ -289,8 +289,12 @@ As an example, consider `Transfer` event from `NEP-17` standard:
 By default, compiler performs some sanity checks. Most of the time
 it will report missing events and/or parameter type mismatch.
 It isn't prohibited to use a variable as an event name in code, but it will prevent
-the compiler from analyzing the event. It is better to use either constant or string literal. 
-The check can be disabled with `--no-events` flag.
+the compiler from analyzing the event. It is better to use either constant or string literal.
+The check can be disabled with `--no-events` flag. It isn't prohibited to use ellipsis
```

Comment:
BTW, we better leave `This check and conversion can be disabled with `--no-events` flag` to the end.